src/backend/updater.ts

```diff
@@ -7,10 +7,10 @@ import { logError, LogPrefix, logInfo } from './logger/logger'
 import { isOnline } from './online_monitor'
 
 autoUpdater.autoDownload = true
-autoUpdater.autoInstallOnAppQuit = true
+autoUpdater.autoInstallOnAppQuit = false
 
-// check for updates every 12 hours
-const interval = 1000 * 60 * 60 * 12
+// check for updates every 6 hours
+const interval = 1000 * 60 * 60 * 6
```

Comment:
is there a downside to making this shorter? might be useful if we need to get out an update fast. maybe 1 hr?

Reply:
No reason, I can make it shorter